test/benchmark/BenchmarkSmallIT.java

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.reasoner.benchmark;
+
+import com.vaticle.typedb.core.TypeDB;
+import com.vaticle.typedb.core.common.parameters.Arguments;
+import com.vaticle.typedb.core.common.parameters.Options;
+import com.vaticle.typedb.core.common.parameters.Options.Database;
+import com.vaticle.typedb.core.concept.Concept;
+import com.vaticle.typedb.core.concept.answer.ConceptMap;
+import com.vaticle.typedb.core.concept.thing.Entity;
+import com.vaticle.typedb.core.concept.thing.Relation;
+import com.vaticle.typedb.core.concept.type.EntityType;
+import com.vaticle.typedb.core.concept.type.RelationType;
+import com.vaticle.typedb.core.database.CoreDatabaseManager;
+import com.vaticle.typedb.core.test.benchmark.generation.DiagonalGraph;
+import com.vaticle.typedb.core.test.benchmark.generation.PathTreeGraph;
+import com.vaticle.typedb.core.test.benchmark.generation.TransitivityChainGraph;
+import com.vaticle.typedb.core.test.benchmark.generation.TransitivityMatrixGraph;
+import com.vaticle.typedb.core.test.integration.util.Util;
+import com.vaticle.typeql.lang.TypeQL;
+import com.vaticle.typeql.lang.query.TypeQLMatch;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static com.vaticle.typedb.core.common.collection.Bytes.MB;
+import static junit.framework.TestCase.assertEquals;
+
+public class BenchmarkSmallIT {
+    private static final String database = "reasoner-benchmark-small";
+    private static final Path dataDir = Paths.get(System.getProperty("user.dir")).resolve("reasoner-benchmark-small");
+    private static final Path logDir = dataDir.resolve("logs");
+    private static final Database options = new Database().dataDir(dataDir).reasonerDebuggerDir(logDir)
```

Comment:
for all these tests - do we need explanation on by default?

Reply:
Good catch.
We don't.